picocolors.js

```diff
@@ -1,60 +1,60 @@
 let tty = require("tty")
 
 let isColorSupported =
-	!("NO_COLOR" in process.env || process.argv.includes("--no-color")) &&
-	("FORCE_COLOR" in process.env ||
+	!process.env.NO_COLOR &&
```

Comment:
It has a different meaning. Empty string will not lead to `false`, but it leads to `true` before.